Added My Simple Wishlist to Apps/Websites

g00dv1n opened the pull request:
### `General`
> ✏️ Mark the necessary items without changing the structure of the PR template.

- [x] Pull request template structure not broken

### `Type`

> ℹ️  What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Fix
- [x] Feature 

### `Checklist`

> ℹ️  Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md).

> 👉  _Put an `x` in the boxes that apply._

- [x] Title as described
- [x] Make sure you put things in the right category!
- [x] Always add your items to the end of a list

#### `Open Source`

- [ ] Link description does not contain a link to an author / third-party resource
- [ ] The documentation (README) contains a description of the project, illustration of the project with a demo or screenshots and a CONTRIBUTING section
- [ ] The documentation is in English.
- [ ] The project is active and maintained.
- [ ] The project accepts contributions.
- [ ] Not a commercial product

#### `Apps/Websites`

- [ ] The website is available without errors or ssl certificate problems, and load in a reasonable amount of time.
- [ ] The website is using vuejs intensively. It should detect vue with [vue-devtools](https://github.com/vuejs/vue-devtools).
  > If you cannot detect vue with `vue-devtools` due to work at non public pages (e.g. for enterprise website), you can send Pull Request with screenshot that detected it.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.
- [ ] A commercial product using Vue, provided that guests could reasonably check out how Vue was used (i.e. A headless CMS which uses Vue for the Admin/editor Area and offers a free tier).

g00dv1n commented:
Hey there!
This is a browser extension, so I've added a link to the landing page that has links to all stores (Chrome, Firefox, etc)

Also, I added separate instances of Dev tools into the extension to create screenshots proof (because when you create an extension you can't use dev tools as an extension)

<img width="1301" alt="Screenshot 2021-05-05 at 22 00 08" src="https://user-images.githubusercontent.com/4170706/117868819-b79e8f00-b2a2-11eb-8212-e56f563c139a.png">
<img width="502" alt="Screenshot 2021-05-05 at 22 00 42" src="https://user-images.githubusercontent.com/4170706/117868826-b9685280-b2a2-11eb-9cf2-b5cecda52919.png">

Scrum reviewed (changes requested):
## Checklist

- [ ] git: pull request template structure not broken
- [ ] git: pull request template not selected `type`
- [ ] git: pull request template not checked `Checklist`
- [ ] git: empty commit
- [ ] git: link not to repository
- [ ] git: link description contains a link to the author / third-party resource
- [ ] git: broken PR
- [ ] git: no conflicts resolved
- [ ] git: doublicate
- [ ] time: not updated for more than 6 months
- [ ] category: not in its category
- [ ] lists: not the last on the list
- [ ] docs: no documentation
- [ ] docs: non-english documentation
- [ ] project: commercial product
- [ ] project: dead project
- [ ] project: does not accept donat
- [ ] site: no link to the site
- [ ] site: site with errors
- [x] site: not actively using vuejs
- [ ] site: not original
> Follow the rules [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md)

Scrum commented:
> Hey there!
> This is a browser extension, so I've added a link to the landing page that has links to all stores (Chrome, Firefox, etc)
> 
> Also, I added separate instances of Dev tools into the extension to create screenshots proof (because when you create an extension you can't use dev tools as an extension)

These screenshots do not give an understanding that this was done from the site

g00dv1n commented:
> > Hey there!
> > This is a browser extension, so I've added a link to the landing page that has links to all stores (Chrome, Firefox, etc)
> > Also, I added separate instances of Dev tools into the extension to create screenshots proof (because when you create an extension you can't use dev tools as an extension)
> 
> These screenshots do not give an understanding that this was done from the site

Hey!

These screenshots related to the extension itself not for the landing page (the extension built with Vue, landing page not). The only reason why I put the link to the landing page, because the extension stored on different stores, Chrome Web Store, Firefox Addons, etc. And on the landing page, we have links to all stores (https://productstories.pro/my-simple-wishlist/#add).

If such approach not allowed, I can change the link directly to Chrome Web Store. What do you think?

Scrum commented:
> If such approach not allowed, I can change the link directly to Chrome Web Store. What do you think?

hi yes, it is better to change the link to the application itself otherwise it may seem like an advertisement / promotion.
